utilities/tests/ordered-multimap-test.metta

```diff
@@ -40,6 +40,22 @@
 ! (assertEqual (MultiMap.checkValue d (ConsMMap (1 a) (ConsMMap (2 b) (ConsMMap (3 c) (ConsMMap (4 d) NilMMap))))) True)
 ! (assertEqual (MultiMap.checkValue f (ConsMMap (1 a) (ConsMMap (2 b) (ConsMMap (3 c) NilMMap)))) False)
 
+;; Testcase for Map.values
+! (assertEqual (MultiMap.values (ConsMultiMap (1 a) (ConsMultiMap (2 b) (ConsMultiMap (3 c) NilMMap)))) (a b c))
```

Comment:
When building maps, we use `ConsMMap` as a constructor instead of the `ConsMultiMap` so change it accordingly.